documentation/modules/exploit/linux/local/bpf_privilege_escalation.md

```diff
@@ -0,0 +1,13 @@
+## Summary
+There is an integer-overflow-to-buffer-overflow vulnerability in the bpf functions introduced in 4.19 and affect up to 4.20-rc3 leading to LPE, running the exploit gives a root shell in a custom compiled 4.19 or 4.20-rc3 system.
+
+## Verification Steps
+
+1.  Start `msfconsole`
+2.  Get a session
+3.  `use exploit/linux/local/bpf_privilege_escalation`
+4.  `set SESSION <ID>`
+5.  Verify you get a new root session
+
+## Source
```

Comment:
This reference is included in the module. It can be removed from here.

Add a `## Scenarios` section with an example execution of the module.